spec/models/provider_spec.rb

```diff
@@ -248,4 +260,29 @@
       expect(Provider.opted_in).to match_array([opted_in_provider])
     end
   end
+
+  describe '#updated_changed_at' do
+    let(:provider) { create(:provider, changed_at: 1.hour.ago) }
+
+    it 'sets changed_at to the current time' do
+      Timecop.freeze do
+        provider.update_changed_at
+        expect(provider.changed_at).to eq Time.now.utc
+      end
+    end
+
+    it 'sets changed_at to the given time' do
+      timestamp = Time.now.utc
```

Comment:
is this at risk of co-incidental pass? maybe use 1 hour ago or something to differentiate from now